### PR TITLE
Add webpack template

### PR DESCRIPTION
**Pull request by rebel216**
Set up the repository and tools.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
```
Comment (MeqdamAlqudah) [inline dist/index.html:4]: - [ ]  Please pull request should have a descriptive title telling you what changes are inside.
- [ ] Please pull request should have a short summary describing introduced changes in more detail.
image

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (MeqdamAlqudah) [inline dist/index.html:6]: - [ ] Please record a video presenting the features of the project. The maximum length of the video is 5 minutes.

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <title>The Food App</title>
+    <script defer src="main.js"></script></head>
+    <body>
+        <div class="container">
+            <header>
```
Comment (MeqdamAlqudah) [inline dist/index.html:11]: - [x] Please make sure to follow the design for the comment popup i.e the close button the comments position

| Your design | Mockup|
| ---------------- | --------- |
| image | image |

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @rebel216 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:+1: Readme file


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 3 above)
Comment (Mihndim2020) [inline dist/index.html:11]: - [ ] As requested by the previous code reviewer, which is also a requirement for this project, please ensure the implementation of your comment popup follows the [design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). Please make sure to follow the design for the comment popup i.e the close button the comments position

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments under the review._

Please do consider, as requested by the previews reviewer:
- [ ] Please pull request should have a descriptive title telling you what changes are inside.
- [ ] Please pull request should have a short summary describing introduced changes in more detail.
- [ ] Please record a video presenting the features of the project. The maximum length of the video is 5 minutes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**